toolkit/components/uniffi-bindgen-gecko-js/src/templates/js/Error.jsm

```diff
@@ -0,0 +1,31 @@
+{%- let error = ci.get_error_definition(name).unwrap() %}
+{%- let string_type = Type::String %}
+{%- let string_ffi_converter = string_type.ffi_converter() %}
+
+{% if error.is_flat() %}
+class {{ error.name().to_camel_case() }} extends Error {}
```

Comment:
I'll factor out the `name().to_camel_case()` I use it everywhere

Reply:
What do you think about using a `ErrorJSExt` extension trait with a `nm()` function for this?